easybuild/easyconfigs/p/ParaView/ParaView-5.1.2-intel-2016b-mpi.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'ParaView'
+version = '5.1.2'
+versionsuffix = '-mpi'
+
+homepage = "http://www.paraview.org"
+description = "ParaView is a scientific parallel visualizer."
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % download_suffix]
+sources = ["ParaView-v%(version)s.tar.gz"]
+
+dependencies = [
+    ('X11', '20160819'),
+    ('Mesa', '12.0.1'),
+    ('libGLU', '9.0.0'),
+    ('Qt', '4.8.7'),
```

Comment:
yes, building ParaView with Qt5 should be possible but all official releases are still with 4.x
Last time I tried 5.x it gave trouble.